integration-tests/tx-rollup-l2-address.spec.ts

```diff
@@ -4,15 +4,15 @@ import { UnitValue } from '@taquito/michelson-encoder';
 
 CONFIGS().forEach(({lib, setup, protocol}) => {
   const tezos = lib;
-  const jakartanet = (protocol === Protocols.PtJakart2) ? test : test.skip;
+  const limanet = (protocol === Protocols.PtLimaPtL) ? test : test.skip;
   const mondaynet = protocol === Protocols.ProtoALpha ? test : test.skip;
 
   describe(`jakarta contract originations and method calls to test the type of tx_rollup_l2_address`, () => {
     beforeEach(async (done) => {
       await setup();
       done();
     })
-    jakartanet(`Originate a contract with a hex string type tz1 & tz4 in initial storage tz4 & string`, async (done) => {
+    limanet(`Originate a contract with a hex string type tz1 & tz4 in initial storage tz4 & string`, async (done) => {
```

Comment:
Should this test run on all testnets? Using `it` rather than `limanet`?
I am not too sure why they were not running on Kathmandu..